modules/configuring-delegated-image-scanning.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+:_content-type: PROCEDURE
+[id="configuring-delegated-image-scanning_{context}"]
+= Configuring delegated image scanning
+
+A new delegated registry configuration specifies the registries from which image scans are to be delegated. For images observed by Sensor, this configuration allows you to delegate scans from no registries (default), all registries, or specific registries. To enable delegation of scans using the `roxctl` CLI, the Jenkins plugin, or the API, you must also specify a destination cluster and source registry.
+
+
+.Prerequisites
+
+* Scanner-slim must be installed in the secured cluster to scan images.
++
+[NOTE]
+====
+Enabling Scanner-slim is supported on {ocp} and Kubernetes secured clusters.
+====
+
+.Procedure
+
+. In the {product-title-short} portal, navigate to *Platform Configuration -> Clusters*.
+. In the *Clusters* view header, click *Manage delegated scanning*.
+. In the *Delegated Image Scanning* page, provide the necessary information:
+** *Delegate scanning for*: Choose the scope of the image delegation. Select one of the following options:
+*** None (default): This option specifies that no images are scanned by the secured clusters, except for images from the {ocp} integrated registry.
+*** All registries: This option indicates that all images are scanned by secured clusters.
+*** Specified registries: This option specifies which images should be scanned by secured clusters based on the registries list.
+
+** *Select default cluster to delegate to*: From the drop-down list, select the name of the default cluster that will process the scan requests coming from the command-line interface (CLI) and API. This is optional and you can select `None` if required.
+
+** (Optional) Click *Add registry*. Specify the source registry and destination cluster details. You can select the destination cluster as `None` if the scan requests are not coming from the CLI and API. You can add more than one source registry and destination cluster if required.
```

Comment:
Should these level 2 list items be ordered instead of unordered? If these different options are all on the same page then maybe it wouldn't matter, but I'm not completely sure.

Reply:
All of them are on the same page.